code/modules/projectiles/guns/ballistic.dm

```diff
@@ -137,7 +137,7 @@
 	AddElement(/datum/element/weapon_description, attached_proc = .proc/add_notes_ballistic)
 
 /obj/item/gun/ballistic/fire_sounds()
-	var/frequency_to_use = sin((90/magazine?.max_ammo) * get_ammo())
+	var/frequency_to_use = sin((90/(magazine?.max_ammo+1)) * get_ammo())
```

Comment:
make it be 1 if there's no max ammo instead

Reply:
90/(magazine?.max_ammo || 1)

Reply:
Isn't this what `max()` is for?

`sin(max(((90/(magazine?.max_ammo+1)) * get_ammo()), 90)`